README.md

```diff
@@ -251,6 +251,19 @@ This validator can also be reused when generating routes:
 router = SpecRouter(specs, validators=[MyValidator])
 ```
 
+## Development
+
+You will need:
+
+- Python 3.10
```

Comment:
3.7 - 3.11 should be fine (it's specified as `>=3.7,<4.0.0`)

Reply:
3.11 broke for me